projects/demo/src/modules/components/avatar/examples/6/index.ts

```diff
@@ -0,0 +1,22 @@
+import {Component, Inject} from '@angular/core';
+import {DomSanitizer} from '@angular/platform-browser';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+
+@Component({
+    selector: 'tui-avatar-example-6',
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    changeDetection,
+    encapsulation,
+})
+export class TuiAvatarExample6 {
+    readonly waterplea = 'https://avatars.githubusercontent.com/u/11832552?v=4';
+    readonly wrongUrl = 'https://google.com/images/not-found.png';
+
+    readonly fallback = this.sanitizer.bypassSecurityTrustHtml(
+        '<svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em"><g xmlns="http://www.w3.org/2000/svg"><svg x="50%" y="50%" width="1em" height="1em" overflow="visible" viewBox="0 0 16 16"><svg x="-8" y="-8"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round"><path d="M20 21v-2a4 4 0 0 0-4-4H8a4 4 0 0 0-4 4v2"/><circle cx="12" cy="7" r="4"/></svg></svg></svg></g></svg>',
```

Comment:
Should we simplify this example ? 

```html
<tui-avatar
    class="avatar"
    [avatarUrl]="waterplea"
></tui-avatar>

<tui-avatar
    class="tui-space_left-3 avatar"
    fallback="tuiIconUserLarge"
    [avatarUrl]="wrongUrl"
></tui-avatar>
```

Or does everything exist on purpose ?

Reply:
fixed